src/main/resources/assets/api-docs.yaml

```diff
@@ -161,6 +161,29 @@ paths:
         500:
           description: Internal Server Error
 
+  /match/v3:
+    post:
+      summary: Match DataUse Purpose and Consent V3
+      description: Determine if a Research Purpose and Consent logically match.
+      parameters:
+        - name: body
+          in: body
+          description: Research Purpose and Dataset DataUse pair.
+          required: true
+          schema:
+            $ref: '#/definitions/DataUseMatchPairV3'
```

Comment:
I don't think this or the other ref exist in Swagger?

Reply:
It does not reference anything, so it is needed.